projects/valtimo/components/src/lib/components/form-io/form-io-builder.component.ts

```diff
@@ -27,7 +30,23 @@ export class FormioBuilderComponent implements OnInit {
   // eslint-disable-next-line @angular-eslint/no-output-native
   @Output() change: EventEmitter<any> = new EventEmitter();
 
-  constructor() {}
+  readonly currentLanguage$ = this.translateService
+    .stream('key')
+    .pipe(map(() => this.translateService.currentLang));
+
+  readonly formioOptions$: Observable<{language: string, i18n: any }> = this.currentLanguage$.pipe(
+    map(language =>  {
+      const formioTranslations = this.translateService.instant('formioTranslations');
+      return typeof formioTranslations === 'object' ? {
+        language,
+        i18n: {
+          [language]: formioTranslations
+        }
+      } : null;
```

Comment:
it doesn't break when null is returned?

Reply:
No null is also valid (won't add any options) and will still work without any warning or error.